generators/app/index.js

```diff
@@ -11,6 +11,21 @@ module.exports = class extends Generator {
     super(args, opts);
   }
 
+  initializing() {
+    if (this.config.existed) {
+      this.log('A Chisel project already exists in this folder.');
+      this.log(
+        'If you are trying to setup existing project check out documentation' +
+          ' at https://www.getchisel.co/docs/setup/wordpress/'
+      );
+      this.log(
+        'If process of generating project was interrupted and you would like' +
```

Comment:
Again some articles and fixing typo in recommend.

_If the process of generating project was interrupted and you would like to continue, we recommend cleaning up the directory and starting again._